homework5.2/homework5.2/Program.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace homework5._2
+{
+    class Program
+    {
+        static void Main()
+        {
+            string word1 = "клоун";
+
+            string word2 = word1.Substring(0, 1) + word1.Substring(3, 1) + word1.Substring(1, 2) + word1.Substring(4, 1);
```

Comment:
Если нужна только одна буква, проще ее взять по индексу напрямую
```suggestion
            string word2 = word1[0] + word1[3] + word1.Substring(1, 2) + word1[4];
```